Documentation/compilation.rst

```diff
@@ -176,7 +209,13 @@ Get your executable with
 
 .. code-block:: bash
 
-   scons -j 32 buildVariablesFile=build/options/supermuc_mac_cluster.py
+   scons -j 4 buildVariablesFile=build/options/supermuc_mac_cluster.py
```

Comment:
Here (and in general): why -j4?
Maybe would be good idea to use placeholder variable for this

Reply:
Completely agree with you. It was setting from my laptop.  I'll fix the script and push updates